x-pack/plugins/lens/public/indexpattern_datasource/fields_accordion.tsx

```diff
@@ -86,7 +94,22 @@ export const InnerFieldsAccordion = function InnerFieldsAccordion({
       id={id}
       buttonContent={
         <EuiText size="xs">
-          <strong>{label}</strong>
+          <p>
+            <strong className={titleClassname}>{label}</strong>
+            {!!helpTooltip && (
+              <EuiIconTip
+                aria-label={helpTooltip}
+                type="questionInCircle"
+                color="subdued"
+                size="s"
+                position="right"
+                content={helpTooltip}
+                iconProps={{
+                  className: 'eui-alignTop',
+                }}
+              />
+            )}
+          </p>
```

Comment:
Please remove the wrapping `p` element. It is not necessary and it is invalid HTML5 to have it as a child of a `button`.